egret/viz/generate_graphs.py

```diff
@@ -396,7 +397,7 @@ def _plot_generation_stack_components():
             reserves_by_hour += reserves_available
     
     if reserve_requirements_by_hour is not None:
```

Comment:
I think this needs to be changed to `reserve_requirements_array` to fix the NameError